### PR TITLE
Travis

### PR DESCRIPTION
**Pull request by FortisFortuna**

### CODE REVIEW

File: src/addresses/Fraxtal_Mainnet.sol [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    address internal constant FRAXSWAP_FRAX_FXS_PAIR = 0xb4dA8dA10ffF1F6127ab71395053Aa1d499b503F;
+    address internal constant FRAXSWAP_FRAX_SFRXETH_FARM = 0xe402a39F788f90607A50254fAf56316E6a78231A;
+    address internal constant FRAXSWAP_FRAX_SFRXETH_PAIR = 0xEBD293F2173082320d88533316F5964298dE316E;
+    address internal constant FRAXSWAP_FRAX_WFRXETH_FARM = 0xfbf1d253FcAA3cE13187dBD5B8610C15Cc8241c7;
+    address internal constant FRAXSWAP_ROUTER = 0x39cd4db6460d8B5961F73E997E86DdbB7Ca4D5F6;
+    address internal constant FRAXSWAP_ROUTER_MULTIHOP = 0x67E04873691258950299Bd8610403D69bA0A1e10;
+    address internal constant FRAXSWAP_WFRX_ETH_PAIR = 0x4527bcEd9d41706D1436507e9a6e354d3FF44ff9;
+}
+
+library Misc {
```
Comment (ThunderDeliverer) [inline src/addresses/Fraxtal_Mainnet.sol:95]: These are Fraxtal mainnet `BalanceChecker` and `DelegationRegistry` addresses:

```suggestion
library Misc {
    address internal constant BALANCE_CHECKER = 0x219356ef2f11a314Dd0D8bfd7f4B0B951Cb16c3a;
    address internal constant DELEGATION_REGISTRY = 0x098c837FeF2e146e96ceAF58A10F68Fc6326DC4C;
```

File: src/addresses/Fraxtal_L2_Devnet.sol [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+library Misc {
+    address internal constant FRAXCHAIN_ADMIN = 0x8b1E899ec5d51387a41Ee68261149F7d3Dfd7160;
+    address internal constant L2_STANDARD_BRIDGE = 0x4200000000000000000000000000000000000010;
+    address internal constant PROXY_ADMIN = 0xfC00000000000000000000000000000000000009;
+}
+
+library Multisigs {
+    address internal constant PROXY_ADMIN = 0xfC00000000000000000000000000000000000009;
+    address internal constant COMPTROLLER = 0x8b1E899ec5d51387a41Ee68261149F7d3Dfd7160;
```
Comment (0xalex88) [inline src/addresses/Fraxtal_L2_Devnet.sol:37]: This is not a msig, it's an EOA, I think it's ok to leave it there tho, lmk what you think

**Comment by 0xalex88** [comment]
I've added the remaining addresses and moved things around to be consisted between chains

File: src/addresses/Fraxtal_L2_Devnet.sol (hunk 2 above)
Comment (FortisFortuna) [reply src/addresses/Fraxtal_L2_Devnet.sol:35]: Will leave it alone

**Pull request merged**